src/parser/jobs/mnk/modules/RiddleOfFire.tsx

```diff
@@ -1,198 +1,257 @@
+			return 0
+		}
+		return (windows[windowIndex].start - windows[windowIndex].start) / 1000
+	}
+}
 
-		if (action == null) { return }
+class BrotherhoodEvaluator implements WindowEvaluator {
+	private brotherhood: Action
 
-		// MNK mentors want the oGCDs :angryeyes:
-		if (this.riddle != null) {
-			this.riddle.casts.push(event)
-		}
+	constructor(brotherhood: Action) {
+		this.brotherhood = brotherhood
 	}
 
-	private onGain(event: Events['statusApply']): void {
-		if (this.riddle != null) { return }
+	suggest(windows: Array<HistoryEntry<EvaluatedAction[]>>) {
+		const possibleNumberOfBrotherhoods = Math.floor(windows.length / 2)
+		const brotherhoodUsagesInsideRof = windows.map(previousValue => previousValue.data.filter(value => value.action === this.brotherhood))
+			.filter(value => value.length > 0).length
+		const missedBhInsideRof = possibleNumberOfBrotherhoods - brotherhoodUsagesInsideRof
 
-		this.riddle = new Riddle(event.timestamp, this.data)
-		this.riddle.rushing = this.data.statuses.RIDDLE_OF_FIRE.duration >= (this.parser.pull.timestamp + this.parser.pull.duration) - event.timestamp
+		return new TieredSuggestion({
+			icon: this.brotherhood.icon,
+			tiers: BROTHERHOOD_OUTSIDE_ROF_SEVERITY,
+			value: missedBhInsideRof,
+			content: <Trans id="mnk.rof.suggestions.brotherhood.content">Missed {missedBhInsideRof} <ActionLink
+				action="BROTHERHOOD"/> inside <ActionLink action="RIDDLE_OF_FIRE"/> window.
+			</Trans>,
+			why: <Trans id="mnk.rof.suggestions.brotherhood.why"><ActionLink
+				action="BROTHERHOOD"/> has a shorter duration than <ActionLink action="RIDDLE_OF_FIRE"/>.</Trans>,
+		})
+	}
 
-		this.riddleHook = this.addEventHook(
-			filter<Event>()
-				.source(this.parser.actor.id)
-				.type('action'),
-			this.onCast,
-		)
+	output(windows: Array<HistoryEntry<EvaluatedAction[]>>): EvaluationOutput {
+		return {
+			format: 'table',
+			header: {
+				header: <ActionLink showName={false} action={'BROTHERHOOD'}/>,
+				accessor: 'brotherhood',
+			},
+			rows: windows.map((window, i) => {
+				return {
+					actual: this.countBrotherhood(window),
+					expected: this.numberOfBrotherhoodsExpectedInWindow(i, windows),
+				}
+			}),
+		}
 	}
 
-	private onDrop(event: Events['statusRemove']): void {
-		this.stopAndSave(event.timestamp)
+	private countBrotherhood(window: HistoryEntry<EvaluatedAction[]>): number {
+		return window.data.filter(value => this.brotherhood === value.action).length
 	}
 
-	private stopAndSave(endTime: number = this.parser.currentEpochTimestamp): void {
-		if (this.riddle != null) {
-			this.riddle.end = endTime
+	private numberOfBrotherhoodsExpectedInWindow(windowIndex: number, windows: Array<HistoryEntry<EvaluatedAction[]>>): number {
+		// The first window should always have a Brotherhood
+		if (windowIndex === 0) {
+			return 1
+		}
+
+		if (this.lastBrotherhoodActionIsGreaterThanCooldown(windowIndex, windows)) {
+			return 1
+		}
 
-			this.history.push(this.riddle)
+		return 0
```

Comment:
This doesn't really capture the nature of the mistake here - if a person does BH > RoF they'll get a 0/0 even though a normal person would consider this to be a BH + RoF window. If a person completely misaligns BH and it drops before RoF even goes up they'll also get 0/0 when we should be calling it 0/1. 

I'm okay with axing this logic altogether and saving it for a separate module (or a separate PR) but I don't think it should be merged as-is.